library/java/io/envoyproxy/envoymobile/engine/JvmFilterContext.java

```diff
@@ -49,7 +49,7 @@ public void passTrailer(byte[] key, byte[] value, boolean start) {
    * @param endStream,   whether this header block is the final remote frame.
    * @return Object[],   pair of HTTP filter status and optional modified headers.
    */
-  public Object onRequestHeaders(long headerCount, boolean endStream) {
+  public Object onRequestHeaders(long headerCount, boolean endStream, long[] ignored) {
```

Comment:
Do we want to add a doc string (to method's doc string) that would explain why it's ignored?